packages/client/src/components/Location/LocationFinder.tsx

```diff
@@ -27,15 +28,16 @@ const isTrueExactMatch = (
   houseNumberFull &&
   stripString(match?.houseNumberFull) === stripString(houseNumberFull);
 
-const LocationFinder: React.FC<{
-  focus: boolean;
-  matomoTrackEvent: Function;
-  sessionAddress: any; // @TODO replace any with address type.
-  setAddress: Function;
-  setErrorMessage: Function;
-  setFocus: Function;
-  topic: any; //@TODO: Replace it with IMTR-Client's TopicType
-}> = ({
+const LocationFinder: React.FC<
+  {
+    focus: boolean;
+    sessionAddress: any; // @TODO replace any with address type.
+    setAddress: (arg0: boolean) => void;
+    setErrorMessage: (arg0: any) => void;
+    setFocus: (arg0: boolean) => void;
```

Comment:
arg0?

Reply:
It seems you can make this whatever you want. `arg0` came from my linter: `Parameter has a name but no type. Did you mean 'arg0: boolean'?`
That's where it came from.
All these work:
 `setAddress: (arg0: boolean) => void;`
 `setAddress: (bla: boolean) => void;`
 `setAddress: (address: boolean) => void;`

Reply:
Changed it to  `setAddress: (address: boolean) => void;` though